bundles/af-core/src/test/resources/schema/0.14.0/adaptive-form.schema.json

```diff
@@ -811,7 +811,9 @@
             "rules",
             "screenReaderText",
             "tooltip",
-            "visible"
+            "visible",
+            "type",
+            "buttonType"
```

Comment:
Why have we added buttonType here ? When was this added as part of spec ?

Reply:
Ideally shouldn't buttonType be a part of the spec?
Did this for the test case for JSON export, changing this for a simpler check now.